README.md

```diff
@@ -0,0 +1,10 @@
+# familie-ef-soknad-api
+
+## Kjør på localhost: 
+bruk Launcher : ApplicationLocalLauncher
+
+## Hvordan logge inn på localhost:
+http://localhost:[app-port]/[app-contextroot]/local/cookie?redirect=/[app-contextroot]/api
```

Comment:
Stemmer det at app-contextroot skal være to ganger? :)

Reply:
Nope - fikser